requirements.txt

```diff
@@ -2,6 +2,7 @@ BeautifulSoup==3.2.1
 ConcurrentLogHandler==0.9.1
 dateutils==0.6.6
 git+https://github.com/adsabs/ADSPipelineUtils.git@master
+git+https://github.com/adsabs/ADSPipelineMsg.git@master

```

Comment:
this is actually not necessary, adsputils will pull in the correct version - so better to not include it here